src/m365/aad/commands/o365group/o365group-teamify.ts

```diff
@@ -26,22 +26,32 @@ class AadO365GroupTeamifyCommand extends GraphCommand {
   }
 
   public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
-    const data: any = {
-      "template@odata.bind": "https://graph.microsoft.com/beta/teamsTemplates('standard')",
-      "group@odata.bind": `https://graph.microsoft.com/v1.0/groups('${encodeURIComponent(args.options.groupId)}')`
+    const data: any = {  
+      "memberSettings": {
+        "allowCreatePrivateChannels": true,
+        "allowCreateUpdateChannels": true
+      },
+      "messagingSettings": {
+        "allowUserEditMessages": true,
+        "allowUserDeleteMessages": true
+      },
+      "funSettings": {
+        "allowGiphy": true,
+        "giphyContentRating": "strict"
+      }
     };
 
     const requestOptions: any = {
-      url: `${this.resource}/beta/teams`,
+      url: `${this.resource}/v1.0/groups/${encodeURIComponent(args.options.groupId)}/team`,
       headers: {
-        accept: 'application/json;odata.metadata=none'
+        accept: 'application/json'
```

Comment:
Is there any reason for us to retrieve the metadata? If not, let's remove it to avoid sending unnecessary data over the wire.

Reply:
No reason behind it. I'll add modify the accept value and submit the pull request.